Source/MBF.SAMCD.Helpers.pas

```diff
@@ -31,9 +31,46 @@ interface
 
 function ReadCal(Position,Size:byte):longword;
 function DivCeiling(a,b:longint):longint;
+function GetCrc8(const pData: Pointer; nLength: Integer): byte;
+function CheckCRC8(const pData: Pointer; nLength: Integer; CrcValue: byte): Boolean;
 
 implementation
 
+const
+  CRCTable8: array[0..255] of byte = (
```

Comment:
Can we use the hardware CRC of SAMCD?
12.11.3 32-bit Cyclic Redundancy Check 
The CRC Table eats up quite somne space in Flash, if we can get rid of that it would be great. 
Anyway we should put the CRC stuff in MBF.SAMCD.CRC.pas as also other Microcontrollers will have the need/hardware support for CRC functions.

Reply:
Crc : good plan !
Systick is only defined in the file I put it in. The other file does not know Systick.